src/steps/check-marketo-munchkin.ts

```diff
@@ -0,0 +1,35 @@
+import { BaseStep, Field, StepInterface } from '../core/base-step';
+import { Step, RunStepResponse, FieldDefinition, StepDefinition } from '../proto/cog_pb';
+
+export class CheckMarketoMunchkin extends BaseStep implements StepInterface {
+
+  protected stepName: string = 'Check that Marketo Munchkin Tracking Loads';
+  protected stepExpression: string = 'the tracking code for munchkin account id (?<id>[a-zA-Z0-9\-]+) should load';
+  protected stepType: StepDefinition.Type = StepDefinition.Type.VALIDATION;
+  protected expectedFields: Field[] = [{
+    field: 'id',
+    type: FieldDefinition.Type.STRING,
+    description: "Munchkin Account ID associated with the user's Marketo instance (e.g. 460-tdh-945)",
+  }];
+
+  async executeStep(step: Step): Promise<RunStepResponse> {
+    const stepData: any = step.getData().toJavaScript();
+    const id: string = stepData.id;
+
+    // Navigate to URL.
+    try {
+      const actual = await this.client.getFinishedRequests();
+      if (!actual.map(request => request.url).includes('https://munchkin.marketo.net/munchkin.js')) {
+        return this.fail('The munchkin.js script was never requested.');
+      } else if (!actual.map(request => request.url).find(url => url.includes(`https://${id}.mktoresp.com/webevents/visitWebPage`))) {
```

Comment:
May want to `toLowerCase()` the `id`, since Marketo instructions may show this as uppercase, but the URL will likely be lowercase.